mule-extensions-api/src/main/java/org/mule/runtime/extension/api/runtime/operation/OperationExecutor.java

```diff
@@ -19,7 +19,7 @@
  *
  * @since 1.0
  */
-public interface OperationExecutor {
+public interface OperationExecutor<T extends ComponentModel> {
```

Comment:
if you do this , it is no longer an OperationExecutor. Needs to change package too